package.json

```diff
@@ -5,6 +5,7 @@
   "scripts": {
     "dev": "next dev",
     "build": "next build",
+    "build:style": "tailwind build styles/index.css -o styles/output.css",
```

Comment:
added `yarn build:style` command which builds `output.css` that can be then purged. I haven't changed the imports yet since it'll be easier to develop with `index.css` for now.

Reply:
I don't understand the output.css. What's the purpose? I've only ever seen it in our projects.